lib/utils/fileSorting.ts

```diff
@@ -42,7 +42,7 @@ export interface FilesSortingOptions {
  * @param nodes Nodes to sort
  * @param options Sorting options
  */
-export function sortNodes(nodes: readonly INode[], options: FilesSortingOptions = {}): INode[] {
+export function sortNodes(nodes: readonly Node[], options: FilesSortingOptions = {}): Node[] {
```

Comment:
Same here,  breaking and not needed. 
We only need the interface (public) not the private implementation.